tests/js/fixtures/pageFilters.ts

```diff
@@ -13,3 +15,29 @@ export function PageFiltersFixture(params: Partial<PageFilters> = {}): PageFilte
     ...params,
   };
 }
+
+export function GetPageFiltersStorageFixture(params = {}) {
```

Comment:
```suggestion
export function PageFiltersStorageFixture(params = {}) {
```
I don't think we use the Get naming in fixtures